docs/hopsworksai/azure/upgrade.md

```diff
@@ -26,7 +26,7 @@ You need to **Stop** your cluster to start the upgrade process. Once your cluste
 ## Step 2: Add upgrade permissions to your user assigned managed identity
 
 !!! note
-    You can skip this step if you already have the following permissions in your user assigned managed identity:
+    You can skip this step if you already have the following permissions in your user assigned managed identity assigned to your resource group:
     ```json 
```

Comment:
I don't understand what you are adding. The user `assigned managed identity` is not assigned to any resource group, it is assigned to the VMs, but I'm not sure it is helping.

Reply:
yeah, but we add role assignments with two different scopes one for storage account and for upgrade we need to add to resource group scope

Reply:
Then give names to the identity and put a link to where you create and give a name, I think it will be easier to read than what it is assigned to.